### PR TITLE
Side branch

### PR DESCRIPTION
**Pull request by MahmoudBakr23**
What's done in HTML and CSS files:
- .grid-1 > 6 (included its articles)
- the navbar
- the footer

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: main.css [hunk 1]
```diff
@@ -0,0 +1,634 @@
...
+}
+
+.article-body a {
+  text-decoration: none;
+  color: white;
+}
+
+.article-footer a {
+  color: white;
+}
```
Comment (andresporras3423) [inline main.css]: This selector has the same styles of selectors on line 84 and line 507. Don't repeat yourself, in order to minimize code you can group selectors with the same style definition.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,634 @@
...
+.container-article a h1 {
+  color: white;
+  font-size: 3em;
+  text-decoration: underline;
+  width: 60%;
+  margin: 5% auto;
+  line-height: 50px;
+}
+
+.grid-5 h1:hover {
```
Comment (andresporras3423) [inline main.css:276]: This selector has the same styles of selectors on line 288 and line 293. Don't repeat yourself, in order to minimize code you can group selectors with the same style definition.

File: main.css [hunk 3]
```diff
@@ -0,0 +1,634 @@
...
+.css-btn {
+  background-color: #ff5a00;
+}
+
+.web-btn {
+  background-color: #aa1994;
+}
+
+.email-btn {
+  background-color: #0168b8;
```
Comment (andresporras3423) [inline main.css]: Minimize code by grouping selectors with the same style definition

File: main.css [hunk 4]
```diff
@@ -0,0 +1,634 @@
...
+}
+
+footer .first-list li {
+  line-height: 40px;
+  margin: 0 15px;
+}
+
+.second-class {
+  display: flex;
+}
```
Comment (andresporras3423) [inline main.css]: This selector has the same styles of the selector on line 482. Minimize code by grouping selectors with the same style definition.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @MahmoudBakr23 @Abidoyinsola1** 
Good job with your first submission for this project :muscle:  
Your page meets most of the requirements  :clap: :clap: :clap:  
Just a couple of details to improve before you can go ahead:
- [ ] All the elements and element's backgrounds and font colors should be grayscale color (black, white or any shade of gray), this rule also applies to images if the students decide to add any
- [ ] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).

### OPTIONAL

- [ ] it's recommended to use the main tag for the dominant content

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras
(for any question you can contact me in slack as @oscarporras02)

**Review by elmejdki — APPROVED** [review]
# Great job so far @team :clap: :clap:, thank you for implementing the required changes by the other TSE :tophat:
### status: approved 

your project is approved you can merge to master now.

_If you have any concerns you want to tell me about don't hesitate to contact me on slack_

**Pull request merged**